### PR TITLE
Remove similar lines in line detection stage

### PR DESCRIPTION
**Pull request by Mati-AAU**

### CODE REVIEW

**Review by S3JER — CHANGES_REQUESTED** [review]

File: app/src/main/java/dk/scuffed/whiteboardapp/pipeline/Pipeline.kt [hunk 1]
```diff
@@ -48,11 +48,13 @@ internal class Pipeline(context: Context, private val initialResolution: Size) :
             this
         )
 
-        val entirePipeline = fullPipeline(context, cameraXStage, this)
+        //val entirePipeline = fullPipeline(context, cameraXStage, this)
```
[the remark this answers is not in the document]
Comment (Mati-AAU) [reply app/src/main/java/dk/scuffed/whiteboardapp/pipeline/Pipeline.kt]: already done

File: app/src/main/java/dk/scuffed/whiteboardapp/pipeline/stages/bitmap_process_stages/OpenCVLineDetectionStage.kt [hunk 2]
```diff
@@ -20,7 +21,11 @@ internal class OpenCVLineDetectionStage(
     private val bitmapOutputStage: BitmapOutputStage,
     private val threshold: Int,
     private val maxLines: Int,
-    pipeline: IPipeline
+    pipeline: IPipeline,
+    private val rhoResolution: Double = 1.0,
+    private val thetaResolution: Double = Math.PI / 180.0,
+    private val rhoThreshold: Double = 0.0,
+    private val thetaThreshold: Double = 0.0
```
[the remark this answers is not in the document]
Comment (Mati-AAU) [reply app/src/main/java/dk/scuffed/whiteboardapp/pipeline/stages/bitmap_process_stages/OpenCVLineDetectionStage.kt:34]: added docs to the class

File: app/src/main/java/dk/scuffed/whiteboardapp/pipeline/stage_combinations/FullCornerDetectionWithDebugDrawing.kt [hunk 3]
```diff
@@ -39,9 +39,13 @@ internal fun fullCornerDetectionWithDebugDrawing(
...
-        150,
-        200,
-        pipeline
+        325,
+        10,
+        pipeline,
+        3.0,
+        Math.PI / 135,
+        100.0,
+        Math.PI / 15
```
[the remark this answers is not in the document]
Comment (Mati-AAU) [reply app/src/main/java/dk/scuffed/whiteboardapp/pipeline/stage_combinations/FullCornerDetectionWithDebugDrawing.kt:48]: in the class docs now

**Review by S3JER — APPROVED** [review]
LGTM

**Pull request merged**